docs/docs/channels/sms/africas-talking.md

```diff
@@ -0,0 +1,71 @@
+
+## Getting Started
+
+To use Africa's Talking provider in the SMS channel, the first step is to create an Africa's Talking account and add your API key and username to Africa's Talking integration on the Novu platform.
+
+## Create a username
+
+You’ll need to create an application to create a username in Africa’s Talking. To do so, follow the steps below:
+
+- [Sign up](https://account.africastalking.com/auth/register) or [Log in](https://account.africastalking.com/auth/login) to your Africa's Talking account.
+
+- Select the team you want your app to be in.
+
+![Image showing Africa’s talking dashboard and a red arrow pointing to a team with text that says “Click Here”](/img/providers/sms/africas-talking/create-a-username-1.png)
+
+If you're a new user and don't have a team yet, you'll have to do so by clicking on `New Team` and entering your team name. Click on `Save` when you're done.
+
+![Image showing the modal to add a new team on Africa’s Talking dashboard](/img/providers/sms/africas-talking/create-a-username-1-1.png)
+
+- On the page that appears, click on the `Create App` button.
+
+![Image showing the modal to add a new team on Africa’s Talking dashboard](/img/providers/sms/africas-talking/create-an-app-button.png)
+
+- On the pop up that appears, enter your application name, username and select a country. Then click Save.
+
+The `username` is what you will use on the Novu platform.
+
+![Image showing the modal to add a new team on Africa’s Talking dashboard](/img/providers/sms/africas-talking/create-an-app.png)
+
+## Generate API key
+
+To generate a new API key, you can follow these steps:
+
+:::note
+Ensure you have created an app in your team.
+:::
+
+- Click on the app you created.
+
+![Image show Africa’s Talking dashboard with a red arrow pointing to an app name](/img/providers/sms/africas-talking/click-app.png)
+
+- On the page that appears, click on `Settings`(on the menu on your left). This will display a dropdown. Click on `API Key` from the dropdown options.
+
+![Image displaying the Settings menu and the API Key page](/img/providers/sms/africas-talking/settings-menu.png)
+
+- On the page that appears, enter your password and click Generate.
+
+![Image showing page to generate API key on Africa’s Talking platform](/img/providers/sms/africas-talking/generate-api-key.png)
+
+- Copy the API Key generated and paste it in the Novu platform or record it somewhere safe for later use because you will not see it from the dashboard on subsequent visits.
+
+:::note
+Once you've generated your API Key, wait about 3 minutes before testing it.
+:::
+
+## Create an Africa's Talking integration with Novu
+
+- Visit the [Integrations](https://web.novu.co/integrations) page on Novu.
+- Locate **Africa's Talking** under the SMS section and click on the **Connect** button.
+- Enter the `API key`.
+- fill in the `username` field.
+- fill in the `from` field.
```

Comment:
I think you have missed, what value user should enter in from field?
Can it be any random value or we have to first verify this value in provider

For example, in case of AWS SES, we first have to verify send email to use it. We cannot use any random email

Reply:
done